app/src/pages/prompt/PromptVersionDetailsPage.tsx

```diff
@@ -46,7 +46,7 @@ function PromptVersionDetailsPageContent({
         <Flex direction="row" justifyContent="space-between">
           <Flex direction="row" gap="size-100">
             <Heading level={2}>{`Version: ${promptVersion.id}`}</Heading>
-            <TagGroup>
+            <TagGroup aria-label="Prompt Version Tags">
```

Comment:
This prevents a ton of warnings in the console.

This is necessary when you only have 1 child and no label for the group so I didn't want to mark this prop as required.